buildstockbatch/workflow_generator/residential_hpxml.py

```diff
@@ -210,11 +226,21 @@ def create_osw(self, sim_id, building_id, upgrade_idx):
           'retain_eplustbl_htm': False,
           'retain_stdout_energyplus': False,
           'retain_stdout_expandobject': False,
-          'retain_schedules_csv': True
+          'retain_schedules_csv': True,
+          'debug': debug
         }
         server_dir_cleanup_args.update(workflow_args['server_directory_cleanup'])
 
         osw['steps'].extend([
+            {
+                'measure_dir_name': 'HPXMLtoOpenStudio',
+                'arguments': {
+                    'hpxml_path': '../../run/home.xml',
+                    'output_dir': '../../run',
+                    'debug': debug,
+                    'add_component_loads': add_component_loads
+                }
+            },
```

Comment:
Calling `HPXMLtoOpenStudio` after `BuildExistingModel` and `ApplyUpgrade`. The `hpxml_path` is written by `BuildExistingModel` and (optionally) overwritten by `ApplyUpgrade`.